mongo/integration/retryable_writes_prose_test.go

```diff
@@ -88,3 +90,53 @@ func TestRetryableWritesProse(t *testing.T) {
 			"expected error %v, got %v", driver.ErrUnsupportedStorageEngine, err)
 	})
 }
+
+func TestRetryableWritesProseStandalone(t *testing.T) {
```

Comment:
nit: This should be called `TestRetryableWritesProse` and should run on all server kinds. It should have a sub-test that only runs on standalones.

Reply:
done